examples/meta/src/multiclass/support_vector_machine.sg.in

```diff
@@ -12,7 +12,7 @@ Labels labels_test = create_labels(f_labels_test)
 
 #![set_parameters]
 real C = 1.0
-Kernel gauss_kernel = create_kernel("GaussianKernel", log_width=0.0)
+Kernel gauss_kernel = create_kernel("GaussianKernel", width=0.0)
```

Comment:
a width of 0 doesnt make sense. I would set it such that the old values are not changed ... though that might be an irrational number so we will have lots of digits

Reply:
yup, I think there are a few of these. I will have to get all these conversions right: `width = exp(log_width * 2.0) * 2.0`?

Reply:
no, I think in the meta examples, I would just set width to 1, or not at all (and the gaussian kernel meta example can show how to set the bandwidth)
Requires re-generating the data though